static/app/views/eventsV2/homepage.spec.tsx

```diff
@@ -112,7 +112,6 @@ describe('Discover > Homepage', () => {
     // Only the environment field
     expect(screen.getAllByTestId('grid-head-cell').length).toEqual(1);
     screen.getByText('Previous Period');
-    screen.getByText('alpha');
     screen.getByText('event.type:error');
```

Comment:
Did you mean to make these assertions?

Reply:
I meant to make the `alpha` assertion here to assert the environment filter, but it requires a bigger code change and I wanted to keep this small. I didn't realize my alpha flag was actually being detected and not the page filter. I'll update this in another PR but the other assertions are more valuable and they actually test the right thing lol

Reply:
Oh yeah I just meant should these be wrapped in an call to expect? I.e. `expect(screen.getByText('Previous Period')).toBeInTheDocument()`, I'm unsure whether just calling `screen.getByText('Previous Period')` is sufficient for an assertion

Reply:
It technically is because `getByText` throws an error and fails the test if it doesn't find it. It does seem like people want to wrap with `expect` as well so I'll keep that in mind when making other changes